test/functional/basic/basic-e2e-specs.js

```diff
@@ -214,6 +214,13 @@ describe('XCUITestDriver - basics', function () {
       size.width.should.be.a.number;
       size.height.should.be.a.number;
     });
+    it('should be able to get the current window size with Rect', async function () {
+      let size = await driver.getWindowRect();
```

Comment:
`driver` here is a [wd](https://github.com/admc/wd) object, not an instance of `XCUITestDriver`. I don't think `wd` has a method for getting the window rect yet.